dist/gauge.js

```diff
@@ -80,12 +79,12 @@
     var key, out, val;
     out = {};
     for (key in obj1) {
-      if (!hasProp.call(obj1, key)) continue;
+      if (!__hasProp.call(obj1, key)) continue;
       val = obj1[key];
       out[key] = val;
     }
     for (key in obj2) {
-      if (!hasProp.call(obj2, key)) continue;
+      if (!__hasProp.call(obj2, key)) continue;
       val = obj2[key];
       out[key] = val;
     }
```

Comment:
hello , excuse how we can integrate the formaNumber in gauge.js ? to help me show % with the value thank you

Reply:
I am not really gauge developer, I just made pull request with changes I made for myself. I can check about formatNumber when I get the time.

Reply:
thank you very much me too am not really gauge developer but i need juste example how i can put the function gauge.js . thank you a lot